protocols/kad/src/handler.rs

```diff
@@ -771,5 +877,19 @@ fn process_kad_response<TUserData>(
             provider_peers,
             user_data,
         },
+        KadResponseMsg::GetValue {
+            result,
+            closer_peers,
+        } => KademliaHandlerEvent::GetValueRes {
+            result,
+            closer_peers,
+            user_data,
+        },
+        KadResponseMsg::PutValue { key, value } => {
+            KademliaHandlerEvent::PutValue {
```

Comment:
Shouldn't this be a `KademliaHandlerEvent::PutValueRes` and I would think without any contents? Again the libp2p-spec isn't really clear to me, but it doesn't say anything about whether there are any fields present in a `PUT_VALUE` response (or indeed that there is any response at all).

Reply:
We should check other implementations, but, yes, I also doubt that there should be any response.

Reply:
Actually, [there is a response](https://github.com/libp2p/go-libp2p-kad-dht/blob/master/handlers.go#L148)  for any message but the `handleAddProvider` which returns a `nil, nil`